src/components/panel/panel.js

```diff
@@ -381,14 +383,16 @@ MdPanelService.prototype.open = function(opt_config) {
  * and similar properties.
  *
  * @param {string} origTemplate The original template.
+ * @param {!Object} The object passed to mdCompiler.
  * @returns {string} The wrapped template.
  * @private
  */
-MdPanelService.prototype.wrapTemplate_ = function(origTemplate) {
+MdPanelService.prototype.wrapTemplate_ = function(origTemplate, config) {
   var template = origTemplate || '';
+  var panelClass = config['panelClass'] || '';
 
   return '<div class="md-panel-outer-wrapper">' +
-            '<div class="md-panel">' +
+            '<div class="md-panel ' + panelClass + '">' +
```

Comment:
Don't append the CSS class into the template string. While somewhat oblique, this is an XSS risk.